server/rtcd.go

```diff
@@ -299,6 +300,9 @@ func resolveURL(u string, timeout time.Duration) ([]net.IP, string, error) {
 }
 
 func (m *rtcdClientManager) newRTCDClient(rtcdURL, host string, dialFn rtcd.DialContextFn) (*rtcd.Client, error) {
+	// Remove trailing slash if present.
+	rtcdURL = strings.TrimSuffix(rtcdURL, "/")
```

Comment:
Shouldn't this one place be enough, and keep the others as is? 
I mean, it doesn't matter in the end, just curious.

Reply:
Those are two completely different services and URLs (job service and rtcd). And we need to do it a third time since there we support an environment override for `MM_CALLS_JOB_SERVICE_URL`.